### ISSUE TITLE
[Navigation Material] Some bottomSheets "float" after updating to 0.29.0-alpha

### ISSUE THREAD

**Issue by spun** [google/accompanist#1479]
**Description**  

After updating to 0.29.0-alpha, some of my bottom sheets are not anchored to the bottom of the screen. 

I think this only happens if the sheet height is a bit higher than `screenHeight/2`. If the sheet is bigger than that, it starts moving down until it "fixes" itself. It's hard to explain, so I've attached a video of the issue.

https://user-images.githubusercontent.com/1004332/212473531-f1089069-ca53-4723-934c-630390399658.mp4

**NOTE:** The bug is not related to the number of items increasing over time like in my video, that's just a demonstration of the issue occurring only when the sheet reaches a certain height. I get the same result using a fixed number of items or an equivalent Modifier.height value.

**Comment by lilemma** [comment]
I have experienced the same problem

**Comment by jossiwolf** [comment]
Hi, do you have an isolated repro we can check this out with?

**Comment by spun** [comment]
Hi jossiwolf. It's easy to make it float, but it depends on the height of the device. That's why I used a `bottomSheet` that adds items over time.

My demo is really simple. I've used one of the [architecture-templates](https://github.com/android/architecture-templates/tree/base) projects (with accompanist 0.29.0-alpha and compose  1.4.0-alpha04), and added this as destination in `Navigation.kt`:

```kotlin
bottomSheet(route = "sheet") {
    var num by remember { mutableStateOf(0) }
    LaunchedEffect(true) {
        repeat(20) {
            num += 1
            delay(700)
        }
    }

    val items = List(num) { "Item $it" }
    Column {
        Text(
            "Num of items: ${items.size}",
            style = MaterialTheme.typography.bodySmall,
            textAlign = TextAlign.Center,
            modifier = Modifier.fillMaxWidth()
        )
        items.forEach { item ->
            NavigationDrawerItem(
                label = { Text(text = item) },
                selected = false,
                onClick = { num += 1 },
            )
        }
    }
}
```
 
That's the code running in my video. Tell me if you need a full project, I can prepare it.

**Comment by jossiwolf** [comment]
Thanks! Please see https://issuetracker.google.com/265610459 for the upstream issue. This will be fixed in the next Compose release.

**Comment by jossiwolf** [comment]
This has been fixed in v0.29.0-alpha.
